app/konnect/org-management/okta-idp.md

```diff
@@ -51,25 +34,57 @@ Create a new application in Okta to manage {{site.konnect_saas}} account integra
     1. For the **Grant type**, ensure the **Authorization Code** checkbox is selected.
     1. For both the **Sign-in redirect URIs** and
     **Sign-out redirect URIs** boxes, enter: `https://cloud.konghq.com/login`
-    1. In the **Assignments** pane, for **Controlled access**, choose your
-    preferred access level for this application. This preferred access level sets the permissions for
-    Okta admins.
+    1. In the **Assignments** pane, for **Controlled access**, choose your preferred access level for this application.
 
 1. Click **Save**.
 
     Leave this page open. You'll need the connection details here to configure your {{site.konnect_saas}} account.
 
+{% endnavtab %}
+{% navtab SAML %}
+### Prepare the Okta application
+
+Create a new application in Okta to manage the {{site.konnect_saas}} account integration.
+
+1. Sign in to your [Okta admin account](https://okta.com/login/).
+2. In the sidebar, click **Applications > Applications**, then click **Create App Integration**.
+1. Select the application type:
+
+    1. For the **Sign-in method**, select **SAML 2.0**.
+    1. Click **Next**.
+
+1. Configure the application:
+    1. In the **General Settings** page, enter a unique name for your application in the **App Name** box. Optionally add a logo in **App Logo** and update **App Visibility**. Click **Next**.
+    1. In the **Configure SAML** page:
+       1. Add placeholder values for the below fields. The actual values will updated after the configuration is updated in Konnect.
+          1. **Single Sign-On URL**: `https://cloud.konghq.com`
```

Comment:
The Single Sign-On URL placeholder should be: `https://global.api.konghq.com/v2/authenticate/{login_path}/saml/acs`
```suggestion
          1. **Single Sign-On URL**: `https://global.api.konghq.com/v2/authenticate/{login_path}/saml/acs`
```